### ISSUE TITLE
htop won't install

### ISSUE THREAD

**Issue by 9mm** [Homebrew/homebrew-core#22518]
**Checklist**

- [x] Confirmed this is a problem with `brew install`ing one, specific Homebrew/homebrew-core formula (not cask or tap) and not every time you run `brew`? If it's a general `brew` problem please file this issue at https://github.com/Homebrew/brew/issues/new. If it's a `brew cask` problem please file this issue at https://github.com/caskroom/homebrew-cask/issues/new. If it's a tap (e.g. Homebrew/homebrew-php) problem please file this issue at the tap.
- [x] Ran `brew update` and retried your prior step?
- [x] Ran `brew doctor`, fixed all issues and retried your prior step?
- [x] Ran `brew gist-logs <formula>` (where `<formula>` is the name of the formula that failed) and included the output link?
- [ ] If `brew gist-logs` didn't work: ran `brew config` and `brew doctor` and included their output with your issue?

To help us debug your issue please explain:
- What you were trying to do (and why)
- What happened (include command output)
- What you expected to happen
- Step-by-step reproduction instructions (by running `brew install` commands)

# Formula additions or changes
To get formulae added or changed in Homebrew please file a [Pull Request](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)
We will close issues requesting formulae changes.

**Comment by 9mm** [comment]
Argh hit command enter on accident

**Comment by ilovezfs** [comment]
correct https://github.com/hishamhm/htop/issues/682#issuecomment-355702934

**Comment by mingtsay** [comment]
FYI the update from https://github.com/hishamhm/htop/issues/682#issuecomment-361925975
TL;DR: There is a bug in 10.13.3 and has been fixed in 10.13.4 public beta.
